api/V1/Device.js

```diff
@@ -75,4 +78,40 @@ module.exports = function(app, baseUrl) {
         }
       });
   });
+
+  /**
+   * @api {get} /api/v1/devices Get list of devices
+   * @apiName GetDevices
+   * @apiGroup Devices
+   *
+   * @apiUse V1UserAuthorizationHeader
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiSuccess {JSON} devices List of devices.
+   *
+   * @apiUse V1ResponseError
+   */
+  app.get(
+      apiUrl,
+      passport.authenticate(['jwt'], {session: false}),
```

Comment:
just so I know, what does the `session: false` here do?

Reply:
It doesn't look like it actually does anything.
https://github.com/themikenicholson/passport-jwt/blob/master/lib/strategy.js#L90
But it was used in the example so probably best to leave it there for now
https://github.com/themikenicholson/passport-jwt#authenticate-requests